lib/workers/repository/finalise/repository-statistics.ts

```diff
@@ -57,6 +59,8 @@ function branchCacheToMetadata({
     branchSha,
     isModified,
     isPristine,
+    isConflicted,
+    isBehindBase,
```

Comment:
Why the need to now log these in metadata?

Reply:
I added it for debugging, will remove once done.